### ISSUE TITLE
Language flags next to the player names

### ISSUE THREAD

**Issue by afuerstenau** [lichess-org/lichobile#2081]
It's hard to talk to someone if you don't speak the same language. I am German, so I can speak German but I can also speak Englisch.

It would be nice to see if we (my opponent and me) are speaking the same language.
![IMG_75AAAEF1DDC7-1](https://user-images.githubusercontent.com/133132/161956187-f6df857d-145e-431d-a466-feef57471009.jpeg)

**Comment by ornicar** [comment]
languages don't have flags, countries do

**Comment by afuerstenau** [comment]
Yep and still in many cases flags can be useful if there is a connection between the country and the language.

I could also live with the name of the language next to the name.
